### PR TITLE
Use single instead of double quotes in Command line docs

### PR DESCRIPTION
**Pull request by hmmbob**
## Proposed change

There is a specific note to use single quotes ( ` ' `) with the command, yet the example used double quotes (` " `) 🤷🏻 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/command_line.markdown [hunk 1]
```diff
@@ -25,7 +25,7 @@ To use your Command binary sensor in your installation, add the following to you
 # Example configuration.yaml entry
 binary_sensor:
   - platform: command_line
-    command: "cat /proc/sys/net/ipv4/ip_forward"
+    command: 'cat /proc/sys/net/ipv4/ip_forward'
```
Comment (frenck) [inline source/_integrations/command_line.markdown:28]: Inside the command itself, yes, it has no value here in this example here.
Maybe the example is wrong/not extensive enough to match that note?

Additionally, this now violates our YAML style guide.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by hmmbob** [comment]
But the note one line down says:

> It’s highly recommended to enclose the command in single quotes ' as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: ''.

I merely adjusted the example to that note, to resolve the discrepancy. All other examples further down are using single quotes.
So.... is the note wrong? Any suggestions to resolve/way forward?

File: source/_integrations/command_line.markdown (hunk 1 above)
Comment (hmmbob) [reply source/_integrations/command_line.markdown:28]: But the note one line down says:

> It’s highly recommended to enclose the command in single quotes ' as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: ''.

I merely adjusted the example to that note, to resolve the discrepancy. All other examples further down are using single quotes too. So.... is the note wrong? Any suggestions to resolve/way forward?

Comment (frenck) [reply source/_integrations/command_line.markdown:28]: YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

As for this particular example & comment. Quotes used inside the command itself, are different for shell scripts when using single our double quotes (shell expansion).

This is not related to the string quotation in YAML.

Comment (hmmbob) [reply source/_integrations/command_line.markdown:28]: Ok, thanks!
I'll just not touch it, stumbled upon it by accident anyway.

**Pull request closed without merging**